Balance/Views/WalletInfoView.swift

```diff
@@ -160,6 +163,8 @@ class WalletInfoView: UIView {
             make.top.equalTo(qrCodeImageView.snp.bottom).offset(20)
             make.centerX.equalToSuperview()
             make.bottom.equalToSuperview().offset(-20)
+            make.leading.equalTo(qrCodeImageView.snp.leading)
```

Comment:
When you're using the same property, you don't need to specify it. So these lines should be:

```
make.leading.equalTo(qrCodeImageView)
make.trailing.equalTo(qrCodeImageView)
```

Reply:
slick! haven't used snapkit before so still getting the API down.